backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -255,7 +256,13 @@ private async void OnIsarMissionUpdate(object? sender, MqttReceivedArgs mqttArgs
 
             if (flotillaMissionRun.IsLocalizationMission())
             {
-                if (flotillaMissionRun.Status != MissionStatus.Successful) { await robotService.UpdateCurrentArea(robot.Id, null); }
+                if (flotillaMissionRun.Status != MissionStatus.Successful)
+                {
+                    await robotService.UpdateCurrentArea(robot.Id, null);
+
+                    signalRService.ReportGeneralFailToSignalR(robot, "Failed Localization Mission", $"Failed localization mission for robot {robot.Name}.");
```

Comment:
Should we have a more specific fail reported? ReportLocalizationMissionFailedToSignalR or something a long that?

Reply:
We used to but then the format that is being sent inside the funtion is the same for all cases, that's why I created this general one that we can call and personalize the content with the second and third field. Otherwise we would have like 3 or more functions with different names that do exactly the same thing